### PR TITLE
feat(edgeless): add fontloader

### LINKED ISSUE #3777: Edgeless mode font loading problem
### What happened?

When I open the client for the first time, the font will change when I create text in edgeless mode and perform a bold operation. After several attempts it returned to normal.

https://github.com/toeverything/AFFiNE/assets/102217452/396440a8-1f7f-4848-afd0-812ce856374c

### Distribution version

macOS ARM 64 (Apple Silicon)

### What browsers are you seeing the problem on if you're using web version?

Chrome

### Relevant log output

_No response_

### Anything else?

_No response_

### Are you willing to submit a PR?

- [ ] Yes I'd like to help by submitting a PR!

### PR DESCRIPTION
**Pull request by doouding**
close #3777

### CODE REVIEW

**Comment by doouding** [comment]
It's hard to debug blocksuite inside AFFiNE locally, so I simulated the slow connection on loading font. This PR should fix the issue.

**Review by doodlewind — APPROVED** [review]

File: packages/blocks/src/page-block/edgeless/edgeless-page-block.ts [hunk 1]
```diff
@@ -1115,10 +1118,26 @@ export class EdgelessPageBlockComponent
     );
   }
 
+  private _initFontloader() {
+    if (!this.fontLoader) this.fontLoader = new FontLoader();
+
+    this._disposables.add(
+      this.fontLoader.slots.loaded.on(font => {
+        if (font !== 'Kalam:n4,n7' || !this.surface) return;
```
Comment (lawvs) [inline packages/blocks/src/page-block/edgeless/edgeless-page-block.ts:1126]: What's `Kalam:n4,n7`?

Comment (doodlewind) [reply packages/blocks/src/page-block/edgeless/edgeless-page-block.ts:1126]: https://fonts.google.com/specimen/Kalam

Comment (doouding) [reply packages/blocks/src/page-block/edgeless/edgeless-page-block.ts:1126]: "n" means `font-style: normal`, "4" and "7" means `font-weight: 400` and `font-weight: 700`

**Pull request merged**